app/api/v1/endpoints/admin.py

```
@@ -228,7 +240,197 @@ async def configure_system(
         logger.error("System configuration failed", error=str(e))
         raise HTTPException(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
-            detail=f"System configuration failed: {str(e)}"
+                    detail=f"System configuration failed: {str(e)}"
+    )
+
+
+@router.get("/credit-pricing")
+async def get_credit_pricing_config(
+    current_user: User = Depends(require_role(UserRole.ADMIN))
```

Ruff (0.12.2)

Unused function argument: `current_user`

(ARG001)